### DISCUSSION TITLE
RSS feeds oldest items at the top, how to reverse the order?

### DISCUSSION THREAD

**Discussion by konomikitten** [lwindolf/liferea#1334] [Q&A]
I've noticed that some feeds do not put the most recent items at the top of their feed and instead have the oldest entries first. I've found no way to tell Liferea to fetch the latest entries from the bottom of these feeds rather than the top. Is this a bug or a violation of the RSS specification?

**Comment by lwindolf** [comment]
Liferea only looks at the timestamps of the feed items and sorts in all items in chronological order.

The only exception being (rare) feeds which items that do not specify a timestamp. In this case Liferea does assign the current time to all new items and has to decide on an order for them. The chosen order is in parsing order. There is no way to change it.

If you encounter such a feed I believe it is best to contact the author to fix it.
